localstack/utils/analytics/usage.py

```diff
@@ -116,6 +119,9 @@ def aggregate_and_send():
     aggregated_payload = aggregate()
 
     publisher = AnalyticsClientPublisher()
-    publisher.publish(
-        [Event(name="ls:usage_analytics", metadata=metadata, payload=aggregated_payload)]
-    )
+    try:
+        publisher.publish(
+            [Event(name="ls:usage_analytics", metadata=metadata, payload=aggregated_payload)]
+        )
+    except Exception as e:
+        LOG.info("Unable to report analytics: %s", e)
```

Comment:
we have a pattern for handling analytics exceptions (which isn't great frankly, but to be consistent):
```suggestion
        if config.DEBUG_ANALYTICS:
            LOG.exception("Error while publishing analytics")
```